orchestration/dagster_orchestration/hca_orchestration/resources/config/target_hca_dataset.py

```diff
@@ -32,19 +32,20 @@ def build_new_target_hca_dataset(init_context: InitResourceContext) -> Optional[
 
     creation_date = datetime.now().strftime("%Y%m%d")
     env = init_context.resource_config['env']
-    target_hca_dataset_name = f"hca_{env}_{hca_project_id.hex.replace('-', '')}__{creation_date}"
+    target_hca_dataset_prefix = f"hca_{env}_{hca_project_id.hex.replace('-', '')}"
```

Comment:
We were always going to create a new dataset if the day rolls over (i.e, a copy on 8/30 would create a dataset, and a subsequent re-copy on 9/1 would create an all-new dataset). This checks for an existing dataset with the needed project_id and re-uses it if present.